### ISSUE TITLE
Ubuntu launchpad builds broken

### ISSUE THREAD

**Issue by springmeyer** [mapnik/mapnik#3060]
```
/usr/bin/clang++-3.5 -o plugins/input/csv/csv_featureset.os -c -std=c++11 -g -O3 -DBOOST_NO_CXX11_SCOPED_ENUMS -Wall -pthread -ftemplate-depth-300 -Wsign-compare -Wshadow -Wno-unknown-pragmas -Wno-unsequenced -O3 -fPIC -DSHAPE_MEMORY_MAPPED_FILE -DMAPNIK_HAS_DLCFN -DBIGINT -DBOOST_REGEX_HAS_ICU -DHAVE_JPEG -DMAPNIK_USE_PROJ4 -DHAVE_PNG -DHAVE_WEBP -DHAVE_TIFF -DLINUX -DMAPNIK_THREADSAFE -DBOOST_SPIRIT_NO_PREDEFINED_TERMINALS=1 -DBOOST_PHOENIX_NO_PREDEFINED_TERMINALS=1 -DBOOST_SPIRIT_USE_PHOENIX_V3=1 -DNDEBUG -Ideps -Ideps/agg/include -Iinclude -I/usr/include -I/usr/include/freetype2 -I/usr/include/postgresql -I/usr/include/gdal plugins/input/csv/csv_featureset.cpp
In file included from plugins/input/csv/csv_datasource.cpp:23:
plugins/input/csv/csv_utils.hpp:144:24: error: no matching function for call to 'min'
    std::size_t size = std::min(file_length, 4000UL);
                       ^~~~~~~~
/usr/bin/../lib/gcc/i686-linux-gnu/4.9/../../../../include/c++/4.9/bits/algorithmfwd.h:367:5: note: candidate template ignored: deduced conflicting types for parameter '_Tp' ('unsigned int' vs. 'unsigned long')
    min(const _Tp&, const _Tp&);
    ^
/usr/bin/../lib/gcc/i686-linux-gnu/4.9/../../../../include/c++/4.9/bits/stl_algo.h:3444:5: note: candidate template ignored: could not match 'initializer_list<type-parameter-0-0>' against 'unsigned int'
    min(initializer_list<_Tp> __l, _Compare __comp)
    ^
/usr/bin/../lib/gcc/i686-linux-gnu/4.9/../../../../include/c++/4.9/bits/algorithmfwd.h:371:5: note: candidate function template not viable: requires 3 arguments, but 2 were provided
    min(const _Tp&, const _Tp&, _Compare);
    ^
/usr/bin/../lib/gcc/i686-linux-gnu/4.9/../../../../include/c++/4.9/bits/stl_algo.h:3439:5: note: candidate function template not viable: requires single argument '__l', but 2 arguments were provided
    min(initializer_list<_Tp> __l)
    ^
1 error generated.
scons: *** [plugins/input/csv/csv_datasource.os] Error 1
```

Seen at https://launchpadlibrarian.net/216718621/buildlog_ubuntu-vivid-i386.mapnik_3.0.0%2Bdev20150909.git.eee1a46-1~vivid1_BUILDING.txt.gz

Assigning to @artemp since I figure this is due to some recent change with CSV plugin. Hopefully an easy fix.

**Comment by artemp** [comment]
fixed in 310dc968ea517df364efeef32304be5a835bfd37

**Comment by springmeyer** [comment]
Yes, fixed. Now launchpad builds are failing with:

```
In file included from plugins/input/csv/csv_datasource.cpp:24:
plugins/input/csv/csv_datasource.hpp:46:10: fatal error: 'boost/geometry/index/rtree.hpp' file not found
#include <boost/geometry/index/rtree.hpp>
         ^
1 error generated.
scons: *** [plugins/input/csv/csv_datasource.os] Error 1
In file included from plugins/input/csv/csv_featureset.cpp:24:
In file included from plugins/input/csv/csv_featureset.hpp:29:
plugins/input/csv/csv_datasource.hpp:46:10: fatal error: 'boost/geometry/index/rtree.hpp' file not found
#include <boost/geometry/index/rtree.hpp>
         ^
1 error generated.
scons: *** [plugins/input/csv/csv_featureset.os] Error 1
```

https://launchpadlibrarian.net/216802632/buildlog_ubuntu-precise-amd64.mapnik_3.0.0%2Bdev20150910.git.e076640-1~precise1_BUILDING.txt.gz

**Comment by artemp** [comment]
@springmeyer - ok, going to fix it

**Comment by artemp** [comment]
fixed in 337525ef60f0abc173940250df94c3e7583dcf1d

**Comment by springmeyer** [comment]
that worked @artemp! Now builds are failing with:

```
	cp -a debian/tmp/usr/lib/mapnik/input/shape.input debian/libmapnik//usr/lib/mapnik/input/
dh_install: libmapnik missing files (usr/lib/mapnik/input/csv*), aborting
make[1]: *** [override_dh_install] Error 255
debian/rules:45: recipe for target 'override_dh_install' failed
make[1]: Leaving directory '/«BUILDDIR»/mapnik-3.0.0+dev20150912.git.ad80c91'
debian/rules:65: recipe for target 'binary-arch' failed
make: *** [binary-arch] Error 2
dpkg-buildpackage: error: fakeroot debian/rules binary-arch gave error exit status 2
```

Which should be fixed by https://github.com/mapnik/debian/commit/c74087f045dd47370771376bd2c274ce891144ab
